.github/workflows/main.yml

```diff
@@ -4,9 +4,9 @@ on:
   push:
     branches:
       - main
-  pull_request:
-    branches:
-      - main
```

Comment:
do we have to disable PR triggers? Are any limitations that necessitate this? i think it's crucial to be able to run the benchmarks on PRs.